packages/react/src/themes/teams/components/Popup/popupContentStyles.ts

```diff
@@ -1,4 +1,4 @@
-import { PopperChildrenProps } from 'react-popper'
+import { PopperChildrenProps } from '../../../../lib/positioner'
```

Comment:
Popup styles shouldn't depend on this, they should depend just on Popup's prop types - otherwise we are breaking encapsulation. It should be sufficient to use just PopupContentProps['placement'] at place where this type was necessary

Reply:
good point, done thanks